hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockInputStream.java

```diff
@@ -235,7 +235,13 @@ protected List<ChunkInfo> getChunkInfos() throws IOException {
                   .getLegacyFactor(pipeline.getReplicationConfig())))
           .build();
     }
-    acquireClient();
+    try {
+      acquireClient();
+    } catch (IOException ioe) {
+      LOG.warn("Failed to acquire client for pipeline " + pipeline +
```

Comment:
Can we use the placeholder syntax "{}" in the log message rather than concatenation? Otherwise LGTM.